pkg/migrate/migrate.go

```diff
@@ -417,6 +264,185 @@ func Migrate(ctx context.Context, w io.Writer, clientset k8sclient.Interface, so
 	return nil
 }
 
+func copyAllPVCs(ctx context.Context, w io.Writer, clientset k8sclient.Interface, sourceSCName string, destSCName string, rsyncImage string, matchingPVCs map[string][]corev1.PersistentVolumeClaim, verboseCopy bool) error {
+	// create a pod for each PVC migration, and wait for it to finish
+	_, _ = fmt.Fprintf(w, "\nCopying data from %s PVCs to %s PVCs\n", sourceSCName, destSCName)
+	for ns, nsPvcs := range matchingPVCs {
+		for _, nsPvc := range nsPvcs {
+			sourcePvcName, destPvcName := nsPvc.Name, newPvcName(nsPvc.Name)
+			_, _ = fmt.Fprintf(w, "Copying data from %s (%s) to %s in %s\n", sourcePvcName, nsPvc.Spec.VolumeName, destPvcName, ns)
+
+			err := copyOnePVC(ctx, w, clientset, ns, sourcePvcName, destPvcName, rsyncImage, verboseCopy)
+			if err != nil {
+				return err
```

Comment:
can you wrap with the pvc name and namespace here?

Reply:
sure, good idea